labgrid/driver/networkusbstoragedriver.py

```diff
@@ -49,6 +51,52 @@ def on_activate(self):
     def on_deactivate(self):
         pass
 
+    @step(args=['filenames', 'target_dir', 'partition'])
+    def copy_files(self, filenames, target_dir, partition=1):
+        if not self.storage.path:
+            raise ExecutionError(
+                "{} is not available".format(self.storage_path)
+            )
```

Comment:
This was copied from NetworkUSBStorageDriver and it's wrong. See #556.

Reply:
#556 was replaced by #573. If that is merged simply decorate `copy_files()` with `@Driver.check_active`.

Reply:
#573 has now been merged.

Reply:
Fixed